README.markdown

```diff
@@ -99,6 +100,18 @@ STP relies on : boost, flex, bison and minisat. You can install these by:
 $ sudo apt-get install cmake bison flex libboost-all-dev python perl minisat
 ```
 
+If your distribution does not come with minisat, STP maintains an updated fork; it can be built as follows:
+
+```
+$ git clone https://github.com/stp/minisat
+$ cd minisat
+$ mkdir build && cd build
+$ cmake ..
+$ make
```

Comment:
I encourage `cmake --build .` instead of `make`, to use whichever build tool is configured.

Reply:
Easy change; I'll make it consistent through-out the whole document.

Reply:
Done!